manifest.json

```diff
@@ -875,6 +875,30 @@
               }
             }
           ]
+        },
+        {
+          "platforms": [
+            {
+              "dockerfile": "src/servercore/ltsc2019/helix/amd64",
```

Comment:
Path should probably be `windowsservercore` instead of `servercore` since that is the brand name.  Makes it consistent with our other repos.